mw-webapp/src/component/label/Label.tsx

```diff
@@ -46,7 +52,16 @@ export const Label = (props: LabelProps) => {
         style={{color: props.label.color, borderColor: props.label.color}}
         className={clsx(styles.label, props.isSmall && styles.small, props.className)}
       >
-        {props.label.name}
+        {props.isEditable ?
+          <EditableText
+            value={props.label.name}
+            onChangeFinish={(name) => props.onChangeValue && props.onChangeValue(name)}
+            isEditable={props.isEditable}
+            placeholder={props.placeholder}
```

Comment:
Do we really need placeholder for label name unput?

Reply:
done